src/core_ocean/shared/mpas_ocn_tracer_sediment_mod.F

```diff
@@ -0,0 +1,235 @@
+!|||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||
+
+ module SEDIMENT_mod
+
+! !MODULE: SEDIMENT_mod
+! !author Zhendong Cao, Mathew Maltrud
+! !date 02/04/2021
+!
+! !DESCRIPTION:
+!
+ !------------------------------------------------------------------------------
+ !
+ !------------------------------------------------------------------------------
+
+! !INPUT PARAMETERS:
+  !-----------------------------------------------------------------------------
+  !   include tracegas and ecosystem parameters
+  !   all variables from these modules have a parm_ prefix
+  !-----------------------------------------------------------------------------
+   use mpas_log
+   use SEDIMENT_parms
+
+   implicit none
+   save
+   private
+
+!-----------------------------------------------------------------------
+!  public/private declarations
+!-----------------------------------------------------------------------
+
+   public :: &
+      SEDIMENT_tracer_cnt,     &
+      SEDIMENT_init,           &
+      SEDIMENT_SurfaceFluxes,  &
+      SEDIMENT_SourceSink
+
+!-----------------------------------------------------------------------
+!  module variables
+!-----------------------------------------------------------------------
+
+   integer (SEDIMENT_i4), parameter :: &
+      SEDIMENT_tracer_cnt = 99
+
+!*****************************************************************************
+
+contains
+
+!*****************************************************************************
+! !IROUTINE: SEDIMENT_init
+! !INTERFACE:
+
+ subroutine SEDIMENT_init(SEDIMENT_indices)
+
+! !DESCRIPTION:
+!  Initialize tracegas tracer module. This involves setting metadata, reading
```

Comment:
Are you describing the sediment tracer module as `tracegas` or something else?

Reply:
This was a typo, should be` Initialize sediment tracer module`. Thanks!